internal/cli/roles_permissions.go

```diff
@@ -0,0 +1,254 @@
+			if err := cli.api.Role.AssociatePermissions(inputs.ID, ps); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	roleAPIIdentifier.RegisterString(cmd, &inputs.APIIdentifier, "")
+	roleAPIPermissions.RegisterStringSlice(cmd, &inputs.Permissions, nil)
+	return cmd
+}
+
+func removeRolePermissionsCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID            string
+		APIIdentifier string
+		Permissions   []string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "remove",
+		Aliases: []string{"rm"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Remove a permission from a role",
+		Long: `Remove an existing permission defined in one of your APIs.
+To remove a permission try:
+
+    auth0 roles permissions remove <role-id> -p <permission-name>`,
+		Example: `auth0 roles permissions remove <role-id> -p <permission-name>
+auth0 roles permissions rm`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := roleID.Pick(cmd, &inputs.ID, cli.rolePickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			if err := roleAPIIdentifier.Pick(cmd, &inputs.APIIdentifier, cli.apiPickerOptionsWithoutAuth0); err != nil {
+				return err
+			}
+
+			var rs *management.ResourceServer
+
+			if len(inputs.Permissions) == 0 {
+				var err error
+				rs, err = cli.pickRolePermissions(inputs.APIIdentifier, &inputs.Permissions)
+				if err != nil {
+					return err
+				}
+			}
+
+			ps := makePermissions(rs.GetIdentifier(), inputs.Permissions)
+			if err := cli.api.Role.RemovePermissions(inputs.ID, ps); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	roleAPIIdentifier.RegisterString(cmd, &inputs.APIIdentifier, "")
+	roleAPIPermissions.RegisterStringSlice(cmd, &inputs.Permissions, nil)
+	return cmd
+}
+
+func (c *cli) apiPickerOptionsWithoutAuth0() (pickerOptions, error) {
+	ten, err := c.getTenant()
+	if err != nil {
+		return nil, err
+	}
+
+	return c.filteredAPIPickerOptions(func(r *management.ResourceServer) bool {
+		u, err := url.Parse(r.GetIdentifier())
+		if err != nil {
+			// We really should't get an error here, but for
+			// correctness it's indeterminate, therefore we return
+			// false.
+			return false
+		}
+
+		// We only allow API Identifiers not matching the tenant
+		// domain, similar to the dashboard UX.
+		return u.Host != ten.Domain
+	})
+}
+
+func (c *cli) pickRolePermissions(id string, permissions *[]string) (*management.ResourceServer, error) {
+	// NOTE(cyx): We're inlining this for now since we have no generic
+	// usecase for this particular picker type yet.
+	var err error
+	rs, err := c.api.ResourceServer.Read(id)
+	if err != nil {
+		return nil, err
+	}
+
+	var options []string
+	for _, s := range rs.Scopes {
+		options = append(options, s.GetValue())
+	}
+
+	prompt := &survey.MultiSelect{
+		Message: "Permissions",
+		Options: options,
+	}
+	survey.AskOne(prompt, permissions)
```

Comment:
We could use our own wrapper, `prompts.AskOne` that has the correct settings and such.

Reply:
And add a new input there, `MultiSelectInput`.

Reply:
`prompts.AskOne` doesn't work with a `MultiSelect`.